bundle-workflow/tests/tests_test_workflow/test_test_args.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import os
+import unittest
+from unittest.mock import patch
+
+from test_workflow.test_args import TestArgs
+
+
+class TestTestArgs(unittest.TestCase):
```

Comment:
Somewhat related to this - we need to add input validation on the test args to ensure version, architecture, etc. inputs are in desired format/options. We could probably address that as part of this PR